.gitignore

```diff
@@ -96,7 +96,6 @@ docs/manual/figures/*.pdf
 docs/manual/figures/*.png
 docs/manual/figures/signature-types-with-canonicalname.dia
 docs/manual/manual.html-e
```

Comment:
It might be nicer to have this here, together with the other `docs` ignores.
How about also moving the other `docs` entries, from line 107, here?

Reply:
Already group them together.